### ISSUE TITLE
Links on sub-package status page broekn

### ISSUE THREAD

**Issue by dstansby** [sunpy/sunpy#2889]
http://docs.sunpy.org/en/stable/code_ref/stability.html - the links in the table don't seem to be working for me, they just go to a readthedocs error page.

**Comment by Cadair** [comment]
yeah the links appear to have `code_ref` in them twice: http://docs.sunpy.org/en/stable/code_ref/code_ref/coordinates.html

**Comment by Cadair** [comment]
This should be a case of removing the `code_ref` from this template https://github.com/sunpy/sunpy/blob/master/docs/dev_guide/stability.rst I think. @nabobalis ?

**Comment by nabobalis** [comment]
Probs.

**Comment by Naman9639** [comment]
I would like to work on this
